setup.py

```diff
@@ -127,14 +130,17 @@
             'invenio_accounts.admin:session_adminview',
         ],
         'invenio_base.api_apps': [
-            'invenio_accounts_rest = invenio_accounts:InvenioAccountsREST',
+            'invenio_accounts_rest_auth = invenio_accounts:InvenioAccountsREST',
         ],
         'invenio_base.apps': [
             'invenio_accounts_ui = invenio_accounts:InvenioAccountsUI',
         ],
         'invenio_base.blueprints': [
             'invenio_accounts = invenio_accounts.views.settings:blueprint',
         ],
+        'invenio_base.api_blueprints': [
+            'invenio_accounts_rest_auth = invenio_accounts.views.rest:create_blueprint',
```

Comment:
There's a bit of an issue here since this is a blueprint entrypoint that depends on the `rest` extras to be installed... There's a way to go around this (but it's not very pretty...):

```python
# invenio_accounts/views/__init__.py

def create_rest_blueprint(app):
    try:
        from .rest import create_blueprint
        return create_blueprint(app)
    except ImpotrError:
        warning(...)

__all__ = ('create_rest_blueprint', )


# in setup.py

...
'invenio_base.api_blueprints': [
    'invenio_accounts_rest_auth = invenio_accounts.views:create_rest_blueprint',
],
...
```

Let's discuss IRL about it.